lib/external/sentry.js

```diff
@@ -133,6 +133,12 @@ const init = (config) => {
   const username = require('os').hostname();
   Sentry.setUser({ username });
 
+  if (isPresent(process.env.SENTRY_TAGS)) {
+    const tags = JSON.parse(process.env.SENTRY_TAGS);
```

Comment:
At https://github.com/getodk/central/pull/317#discussion_r985143621, I was thinking of wrapping the `JSON.parse()` in a try/catch. However, I've ended up thinking that that'd be premature. We don't expose `SENTRY_TAGS` to users. And if there's a problem with `SENTRY_TAGS`, that would indicate a bug that we'd want to know about.